orchagent/dash/dashrouteorch.cpp

```diff
@@ -220,7 +227,7 @@ bool DashRouteOrch::removeOutboundRouting(const string& key, OutboundRoutingBulk
     if (isRouteGroupBound(ctxt.route_group))
     {
         SWSS_LOG_WARN("Cannot remove route from route group %s as it is already bound", ctxt.route_group.c_str());
-        return true;
+        return false;
```

Comment:
Why is this changed?

Reply:
IMO this should have been false to begin with, if route group is bound then we can keep trying to remove it, similar to removing objects w/ nonzero reference counts

Reply:
Yes, 'false' means to not stop trying till route group is unbounded.